lib/internal/fs/streams.js

```diff
@@ -374,9 +404,19 @@ WriteStream.prototype._writev = function(data, cb) {
     size += chunk.length;
   }
 
+  this[kIsPerformingIO] = true;
   fs.writev(this.fd, chunks, this.pos, function(er, bytes) {
```

Comment:
This needs to be an arrow function in order to properly capture `this`. Alternatively use `self`. Seems to be a bit of a mix.

Reply:
I think we use `this` in the other places, so this should probably be an arrow function and the  `self` occurrences replaced by `this`?

Reply:
Yeah, I’ve switched this to use an arrow function like `_write` does